include/fplus/numeric.h

```diff
@@ -135,6 +135,107 @@ X int_power(X base, X exp)
     return base * int_power(base, exp - 1);
 }
 
+template <typename... Conds>
+struct and_ : std::true_type
```

Comment:
@xtofl You're right. This can be done e.g. like so:

```
#include <type_traits>
#include <string>

template <typename... Conds>
struct and_;

template <typename Cond>
struct and_<Cond> : Cond
{
};

template <typename Cond, typename... Conds>
struct and_<Cond, Conds...> : std::conditional<Cond::value, and_<Conds...>, std::false_type>::type
{
};

template <typename... T>
using are_all_pod = and_<std::is_pod<T>...>;

int main()
{
  static_assert(are_all_pod<int, float>::value, "");
  //static_assert(are_all_pod<int, float, std::string>::value, ""); - this will fail compilation
  static_assert(and_<std::true_type>::value, "");
  static_assert(and_<std::true_type,std::true_type>::value, "");
  //static_assert(and_<std::true_type,std::false_type>::value, ""); - this will fail compilation
  //static_assert(and_<std::false_type>::value, "");
  //and_<> a2; - this will fail compilation
}
```

Maybe there's a more elegant way but I cannot think of it right now.